test/TestApp.AspNetCore.2.0/Startup.cs

```diff
@@ -51,7 +50,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<RequestsCollector>();
             services.AddSingleton<DependenciesCollectorOptions>(new DependenciesCollectorOptions());
             services.AddSingleton<DependenciesCollector>();
-            services.AddSingleton<IExportComponent>(Tracing.ExportComponent);
+            services.AddSingleton<IExportComponent>(ExportComponent.NewNoopExportComponent);
```

Comment:
i probably miss something - why do we need to register noop? It doesn't seem right

Reply:
Reverted to original